src/index.spec.ts

```diff
@@ -22,7 +22,9 @@ describe('Package', () => {
 				'fixtures/first/nested/file.md',
 				'fixtures/second/file.md',
 				'fixtures/second/nested/directory/file.md',
-				'fixtures/second/nested/file.md'
+				'fixtures/second/nested/file.md',
+				'fixtures/third/library/a/book.md',
+				'fixtures/third/library/b/book.md'
```

Comment:
Adding new markdown files to fixtures required updating a few unit tests, no other reason for the change.